src/frontend/src/common/components/library/data_table/index.tsx

```diff
@@ -334,9 +334,17 @@ export const DataTable: FunctionComponent<Props> = ({
       );
     }
 
+    // All headers use the base style.header
```

Comment:
You can do this more cleanly with classNames.

import classNames from "classnames";

// All headers use the base style.header
// However, if header needs to show checkbox, we add additional style
// to override certain aspects of header. See style file for more.
const headerClasses = classNames(style.header, {
  [style.headerWithCheckbox]: showCheckboxes,
});

Reply:
neat

Reply:
Cool, thanks for the suggestion!